source/admin/config.yml

```diff
@@ -21,4 +23,217 @@ collections:
+      - {label: "Category", name: "category", widget: "hidden", default: "catalog"}
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Slug", name: "slug", widget: "string"}
+      - {label: "Catalog PDF", name: "pdf", widget: "file", allow_multiple: false}
+      - {label: "Catalog Cover", name: "cover", widget: "image", allow_multiple: false}
+      - label: "Catalog Topic"
+        name: "topic"
+        widget: "relation"
+        collection: "catalog_topics"
+        searchFields: ["title"]
+        valueField: "title"
+        multiple: true
+
+  # Product Collections
+  # Description: 
+  #   Product collections are groupings of products by a an arbitrary concept (ie mermaids)
+  #   These are used to showcase products in a variety of contexts and function akin to tags.
+  - name: "product_collections"
+    label: "Product Collections"
+    folder: "source/_posts/inventory/product_collections"
+    create: true
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    fields: # The fields for each document, usually in front matter
+      - {label: "Published Status", name: "published", widget: "hidden", default: true}     
+      - {label: "Category", name: "category", widget: "hidden", default: "product_collection"}
+      - {label: "Collection Name", name: "title", widget: "string"}
+      - {label: "Product Collection Cover", name: "image", widget: "image", allow_multiple: false}
+
+  # Products
+  # Description: 
+  #   A Product is a single product unit, it is in catalogs and collections.
+  - name: "product"
+    label: "Products"
+    folder: "source/_posts/inventory/products"
+    create: true
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    fields: # The fields for each document, usually in front matter
+      - {label: "Layout", name: "layout", widget: "hidden", default: "product"}
+      - {label: "Published Status", name: "published", widget: "hidden", default: true}     
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Slug", name: "slug", widget: "string"}    
+      - {label: "Featured", name: "featured", widget: "boolean"}
+      - {label: "SKU", name: "sku", widget: "string"}
+      - {label: "Color", name: "color", widget: "string"}    
+      - {label: "Product Image", name: "image", widget: "image", allow_multiple: false}
+      - label: "Catalog"
+        name: "catalog"
+        widget: "relation"
+        collection: "catalogs"
+        searchFields: ["title"]
+        valueField: "title"
+        multiple: true
+      - label: "Product Collection"
+        name: "product_collections"
+        widget: "relation"
+        collection: "product_collections"
+        searchFields: ["title"]
+        valueField: "title"
+        multiple: true
+      - {label: "Product Description", name: "body", widget: "markdown"}
+
+  # Text Blurbs
+  # Description: 
+  #   These are pieces of content that are showcased across the site.
+  - name: "text_blurbs"
```

Comment:
Yea I'd agree - lets maybe just chat re: what approach we want to take? Its also unclear if we need intro blurbs in the new design. I was just putting it there as a temporary stop gap so the text is there and editable. But agree - I think we should just come up with a strategy for the arbitrary text blurbs thats not as ad-hoc. Want to do that in this PR or in a new one? 

I guess also are we sure we want to convert the FAQ? Im not sure if they LIKED the flexibility that the giant markdown afforded, so had left it. But I'd agree a data file could be the best for that.

Reply:
Okay, sounds like a plan - we can just leave this here temporarily and then clean up in the next PR, or when we update the relevant pages.

RE the FAQ, I suggest a yml file just because we could loop through it in the template using any markup we want. Rendering a big blob of markdown, on the other hand, locks us into h3 tags. The experience of editing in the CMS would be more or less identical I think...